backend/routes/api/users.js

```diff
@@ -48,6 +48,21 @@ router.get("/current", requireVolunteerStatus, (req, res) => {
   });
 });
 
+// get all users of given role
+router.get(
+  "/role/:role",
+  requireAdminStatus,
```

Comment:
Can we add Celebrate/Joi middleware to validate that a valid role was passed in? I think we can even check if it fits one of the values in our enum.